app/domain/token_factory.rb

```diff
@@ -1,18 +1,30 @@
 # frozen_string_literal: true
 
+require 'date'
+
 class TokenFactory < Dry::Struct
 
   NoSigningKey = ::Util::ErrorClass.new(
     "Signing key not found for account '{0}'")
 
   attribute :slosilo, ::Types::Any.default{ Slosilo }
 
+  MAXIMUM_AUTHENTICATION_TOKEN_EXPIRATION = 5.hours
```

Comment:
Mind adding a newline after this?
```suggestion
  MAXIMUM_AUTHENTICATION_TOKEN_EXPIRATION = 5.hours
  
```

Reply:
Added